### ISSUE TITLE
loss function dimension problem

### ISSUE THREAD

**Issue by nchos88** [experiencor/basic-yolo-keras#211]
when test custom_loss with dummy data, `pred_box_xy` throw dimension exception. 

data what i use is from 
     `dummy_array = np.zeros((1,1,1,1,TRUE_BOX_BUFFER,4))`

test code is 
` custom_loss(dummy_array , dummy_array)`
and this below throw exception 
`pred_box_xy = tf.sigmoid(y_pred[..., :2] ) + cell_grid`

dimension of `tf.sigmoid(y_pred[..., :2] )` is [ 1 1 1 1 50 2]
and dimension of `cell_grid ` is [ 16 13 13 5 2]

**Comment by experiencor** [comment]
@nchos88 You need to feed in ```true_boxes``` too.

**Comment by nchos88** [comment]
@experiencor You mean , input shape of loss is [batch 13 13 5 5+class] ?

**Comment by experiencor** [comment]
@nchos88 You can search ```true_boxes``` in frontend.py.
